about.html

```diff
@@ -193,6 +193,45 @@ <h2 id="awards"><a href="#awards"> Awards and Prizes </a></h2>
           <li> June 2021: Adoption of Gammapy as Science Analysis Tool package by the <a href="https://www.cta-observatory.org/">CTA Observatory</a> - <a href="https://www.cta-observatory.org/ctao-adopts-the-gammapy-software-package-for-science-analysis/" class="card-link">CTAO Press Release</a></li>
         </ul>
 
+        <br><br>
+        <h2 id="ackno"><a href="#ackno"> Acknowledgements </a></h2>
+        <p>We would like to thank the Astropy, Numpy, Scipy, IPython, Jupyter and Matplotlib communities for providing their packages
+            which are invaluable to the development of Gammapy. We thank the GitHub team for providing us with an
+            excellent free development platform. Thanks to JetBrains making
+            <a href="https://www.jetbrains.com/pycharm/">PyCharm</a> freely available for Gammapy development.
+        </p>
+
+        <p>We would like to deeply thank all the Gammapy developers, experts and also users that have provided feedback
+            and submitted bug reports. And we warmly thank all group leaders of the
+            <a href="https://gammapy.org/team.html#support">supporting institutes</a>.
+        </p>
+
+        <p>We thank also the funding support from various entities:
+            <ul>
+            <li>the Spanish Ministry of Science and Innovation and the Spanish Research State Agency (AEI) under grant
+                PID2019-104114RB-C33/AEI/10.13039/501100011034,</li>
+            <li>the Institute of Cosmos Sciences University of Barcelona (ICCUB, Unidad de Excelencia “María de Maeztu”)
```

Comment:
<li>the French research ministry founding CTAO as Très Grande Infrastructure de Recherche,</li>